src/services/UserService.ts

```diff
@@ -136,8 +136,26 @@ const getUserProfile = async (token: any): Promise<any> => {
   }
 }
 
+const createFacilityLogin = async (payload: any): Promise <any> => {
```

Comment:
Rename this to createFacilityUser as we are not creating userLogin here.

Reply:
Done sir